wordpress-mobile/WordPress-iOS
# While offline, attach media even after upgrading the app

Fixes #12143 

### To test:

1. Go offline
2. Run the app in Xcode
3. Create a post with an image
4. Exit the Editor and save the draft
5. Stop app by stopping Xcode
6. Run the app in Xcode again
7. Check if the image thumbnail appears
8. Open the post again, check that all images are visible

### Additional information

* Before editing a post, the post's content is updated to reflect the correct local URLs - this will only happen with posts with `isFailed = true`
* This also fixes non-image assets, like videos for example
* The `pathForDisplayImage` property now always return a correct path for a local image, otherwise, it will return `nil`

Update release notes:
